app/views/cases/offender_sar/_data_request_request_type.html.slim

```diff
@@ -1,5 +1,5 @@
                 = t :"helpers.label.data_request.request_type.#{data_request.request_type}"
-                - if data_request.display_request_type_note?
+                - if data_request.request_type_note.present?
```

Comment:
This needs to also check if the request_type_note is allowed for the selected request type.